ipi/engine/__init__.py

```diff
@@ -13,6 +13,7 @@
     "simulation",
     "forces",
     "ensembles",
+    "eda",
```

Comment:
could you use something more descriptive?

Reply:
The `EDA` class has not changed name, but is only used "internally", then the name `eda` should not appear anymore in any `__all__` variable or `__init__.py` file.
I created a new `motion` class called `driven_dynamics` for this purpose.